cmd/manager/main.go

```diff
@@ -154,6 +155,11 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err := consolev1.AddToScheme(mgrScheme); err != nil {
+		log.Error(err, "Failed adding openshift/api/console/v1 to scheme")
+		os.Exit(1)
```

Comment:
Do we want a QuickStart failure to prevent the Operator from coming up?

Reply:
this is to add the consolequickstart crds to the kubernetes api schema. Its done in the above code lines too for all custom resources.